code/ship/ship.cpp

```diff
@@ -4125,34 +4885,34 @@ int get_default_player_ship_index()
 // Goober5000 - this works better in its own function
 void ship_set_default_player_ship()
 {
-	int i;
-
 	// already have one
 	if(strlen(default_player_ship))
 		return;
 
+	auto it = Ship_info.cbegin();
```

Comment:
Ok I see this was already rectified but I thought I could clarify here.  The old single 'int i' syntax was done mostly to handle Visual C++ 6, as it does not consider for loops a separate scope, so two for loops each declaring 'int i' would lead to duplicate declaration of i.  It's the only compiler I'm aware of that ever had that issue that we've supported, and if we have indeed officially dropped any and all VC6 support then we can stop catering to that behavior.  So in-loop iterator declaration should be fine, and it should probably be the preferred way going forward.